build-config.js

```diff
@@ -54,6 +51,9 @@ module.exports = {
 
 		image: {
 			toolbar: [ 'imageStyleFull', 'imageStyleSide', '|', 'imageTextAlternative' ]
-		}
+		},
+
+		// UI language. Language codes follow the https://en.wikipedia.org/wiki/ISO_639-1 format.
+		lang: 'pl'
```

Comment:
Why pl not en?

Reply:
Yep. It should be 'en'. I'll give it R- as I'm still working on the main task and I'm using this repo mainly as a battleground for that. I'll clean up everything here once I've done it.